creator/files/migrations/0024_set_old_types_to_oth.py

```diff
@@ -0,0 +1,24 @@
+from django.db import migrations
+from creator.files.models import FileType
+
+
+def forwards_func(apps, schema_editor):
+    File = apps.get_model("files", "File")
+    db_alias = schema_editor.connection.alias
+    # These will be deleted in a future commit, should probably hard-code
+    old_types = ["SHM", "CLN"]
+    File.objects.using(db_alias).filter(file_type__in=old_types).update(
+        file_type=FileType.OTH.value
+    )
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('files', '0023_add_document_types'),
+    ]
+
+    operations = [
+        migrations.RunPython(forwards_func)
```

Comment:
I think this might need the `reverse_code` populated with a no-op in case we need to roll this back for any reason:
https://docs.djangoproject.com/en/3.2/ref/migration-operations/#django.db.migrations.operations.RunPython.noop